### DISCUSSION TITLE
tutorials/client-side-graphql-react/10-mutations-in-action

### DISCUSSION THREAD

**Discussion by giscus** [apollographql-education/content-feedback#138] [course feedback]
# tutorials/client-side-graphql-react/10-mutations-in-action

https://www.apollographql.com/tutorials/client-side-graphql-react/10-mutations-in-action

**Comment by Victoremepunto** [comment]
Nice course ! kudos to the author(s) !

One thing about the "Seeing the results" - you can leverage the Browser's Developer tools "throttling" feature, on the Network tab

- On Chrome-based browsers: Developer Tools, Network tab, use throttling dropdown selector (normally set to "No throttling"), the "slow 3G" preset worked for me.
- On Firefox: Developer Tools, Network tab, use throttling dropdown selector (normally set to "No throttling"), the "GPRS" worked for me

Hope that helps !
